ui/Dockerfile

```diff
@@ -1,12 +1,23 @@
-FROM node:14-alpine3.14 as build
-ENV REACT_APP_NETWORK=gnosis
-WORKDIR /app
-RUN apk update && apk add git && git clone https://github.com/dappnode/eth2-keymanager-frontend-dappnode-temporary.git
-WORKDIR /app/eth2-keymanager-frontend-dappnode-temporary
-RUN npm install --globally react-scripts
-RUN yarn
-RUN yarn run build
+FROM ghcr.io/dappnode/keymanager-ui:0.1.1 as build
 
 FROM nginx:1.21.6-alpine
-COPY --from=build /app/eth2-keymanager-frontend-dappnode-temporary/build/ /usr/share/nginx/html/
+
+# Static build
+COPY --from=build /app/build /usr/share/nginx/html/
+
+# Default port exposure
 EXPOSE 80
+
+# Copy .env file and shell script to container
+WORKDIR /usr/share/nginx/html
```

Comment:
As a suggestion: you could implement this final docker image layer already in the keymanager UI repository. Remove the `COPY .env .` command and warn people to set the envs somehow (throug compose file, docker flags, etc). Finally the `.env.sh` script woul read the envs from the current environment and write them down to the `env-config.js` file.

This way you keep the reduced docker image in the keymanager official docker image. Also the way of using the official docker image would be much simple.